tests/test_views.py

```diff
@@ -4030,8 +4030,9 @@ def test_external_login_confirm_email_get_link(self, mock_link_confirm):
         self.user.save()
         assert not self.user.is_registered
         url = self.user.get_confirmation_url(self.user.username, external_id_provider='orcid', destination='dashboard')
-        res = self.app.get(url, auth=self.auth)
+        res = self.app.get(url)
```

Comment:
I think this is a good fix :)

I took a look at the code for `external_login_confirm_email_get()`, the comment says:

> When users click the confirm link, they are expected not to be logged in. If not, they will be logged out first and
    redirected back to this view. After OSF verifies the link and performs all actions, they will be automatically
    logged in through CAS and redirected back to this view again being authenticated.

I am surprised the test passed before the upgrade. User is expected to arrive at the link without auth. I am still investigating out of curiosity but it should not prevent me from merging this PR. Finally, we should remember to test this during QA.